### ISSUE TITLE
Not support [TOC]

### ISSUE THREAD

**Issue by hhhyde** [bordaigorl/sublime-evernote#154]

**Comment by bordaigorl** [comment]
Thereare two issues preventing this to happen:

 1. the current markdown converter can only produce the TOC when also inserting `id` attributes in headings. This clashes with Evernote's format restrictions (no `id` nor `class` attributes are allowed)

 2. even if we fixed this, Evernote does not support internal links, making a TOC rather useless

I think Evernote is designed to support a workflow where every "project" has its Notebook and the notes are short and to the point.
So a similar effect to a TOC can be obtained by having a single note per section and using tags/notebooks to assemble the sections under an uniform topic. Then you can also add a "TOC" note to the notebook containing links to the other notes (this can be done, see [the wiki](https://github.com/bordaigorl/sublime-evernote/wiki/Getting-Note-Links))

**Comment by jonbra** [comment]
I tried to make a custom TOC by using anchors/internal links using markdown syntax, e.g.
```
[1. Overview](#overview)   

## Overview <a name="overview"></a>
```
But the links doesn't work in the Evernote app. Is this due to the same reasons?

**Comment by bordaigorl** [comment]
Yes. Quite simply, internal links are not supported by any Evernote client I know.

However, let me point out that with a properly configured markdown syntax you can get your sections indexed by ST (<kbd>Ctrl+R</kbd>) and available as a TOC in a quick panel.
This way you can have automatic TOCs in ST without having a textual toc in the note itself.

**Comment by cjh1993129** [comment]
Evernote does  support TOC now！

**Comment by bordaigorl** [comment]
@cjh1993129 is this documented somewhere?

**Comment by 308820773** [comment]
Maybe you can create a fake TOC in the evernote app (can't link).

**Comment by louisw00** [comment]
heaps markdown editors already have this function. check cmdmarkdown or marxico.
